.travis.yml

```diff
@@ -53,6 +58,22 @@ matrix:
       env: [ ARCH=x86_64, BUILD_TYPE=Debug, GENERATOR="Visual Studio 15 2017" ]
     - <<: *windows_vs2017
       env: [ ARCH=x86_64, BUILD_TYPE=Release, GENERATOR="Visual Studio 15 2017" ]
+    - stage: Coverity
+      <<: *linux_gcc8
+      name: Coverity Build
+      env:
+        - BUILD_TYPE=Release
+        - C_COMPILER=gcc-8
+        - CXX_COMPILER=g++-8
+        - USE_SANITIZER=none
+        - COVERITY_SCAN_PROJECT_NAME="$TRAVIS_REPO_SLUG"
+        - COVERITY_SCAN_BRANCH_PATTERN="coverity_scan"
+        - COVERITY_SCAN_NOTIFICATION_EMAIL="$(git log -1 $TRAVIS_COMMIT --pretty=\"%aE\")"
+        - COVERITY_SCAN_BUILD_COMMAND="cmake --build ."
+        - COVERITY_SCAN_BUILD_COMMAND_PREPEND="cov-configure --comptype gcc --compiler gcc-8 --template"
+        # Encrypted COVERITY_SCAN_TOKEN env variable
+        # Generated using `travis encrypt -r eclipse-cyclonedds/cyclonedds COVERITY_SCAN_TOKEN=xxx`
+        - secure: "insert secret here"
```

Comment:
I don't like the secret being in the `.travis.yml` for the aforementioned reason.

Reply:
I will remove the secret and the token can be added in the travis-ci internal variable section. Doing this will break the windows builds see https://travis-ci.community/t/current-known-issues-please-read-this-before-posting-a-new-topic/264/6 so for now I have to postpone this  PR as it will break windows builds.

Reply:
Yeah, I thought about this... can't we export the encrypted version in an environment variable and then call decrypt? Turn it around?